pom.xml

```diff
@@ -38,9 +38,14 @@
 
   <dependencies>
     <dependency>
```

Comment:
Do we still need this dependency - is it a transitive dependency of selenium 3.5.3?

Reply:
Yes. We need to explicitly state it. It does not work otherwise.